src/raftstore/store/store.rs

```diff
@@ -129,6 +135,8 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             store: meta,
             engine: engine,
             sendch: sendch,
+            send_snapshot_count: 0,
```

Comment:
To be more precise, `sending_snapshot_count` is a better name, since these snapshots are just sending. When the snapshot is sent, a status is reported back to store.